tests/test_estimator.py

```diff
@@ -175,20 +195,22 @@ def test_crs_preserved(model_data: ModelData, crs):
 
 @parametrize_model_data(image_types=(np.ndarray,))
 def test_with_non_image_data(model_data: ModelData):
-    """Test that non-image data falls back to the wrapped estimator behavior."""
+    """Test that 1D sample data is correctly handled."""
     _, X, y = model_data
 
     estimator = KNeighborsRegressor().fit(X, y)
     reference_pred = estimator.predict(X)
     ref_dist, ref_nn = estimator.kneighbors(X)
 
     wrapped = wrap(clone(estimator)).fit(X, y)
-    check_pred = wrapped.predict(X)
-    check_dist, check_nn = wrapped.kneighbors(X)
-
-    assert_array_equal(reference_pred, check_pred)
-    assert_array_equal(ref_dist, check_dist)
-    assert_array_equal(ref_nn, check_nn)
+    # 1D arrays must be in (features, samples) shape to match the expected
+    # input
```

Comment:
I had to have *one* nitpicky comment.  It might not be totally obvious to users that X is (samples, features) to begin with, so tacking on a note like "hence the need for the array transform".  I'm fine if you don't include it, though.

Reply:
Thanks, I added some more detail to clarify this!